tests/dummy/app/controllers/index.js

```diff
@@ -4,6 +4,12 @@ import momentFormat from 'ember-moment/computeds/format';
 import momentFromNow from 'ember-moment/computeds/from-now';
 
 export default Ember.Controller.extend({
+  moment: Ember.inject.service(),
+  actions: {
+    changeLocale(locale) {
+      this.set('moment.locale', locale);
```

Comment:
lets make this a method, `this.get('moment').changeLocale(locale)` this is more DDAU, and allows the service to nicely intercept if needed.